### PR TITLE
User info

### PR DESCRIPTION
**Pull request by kikavlntn**
userInfo component and story created #12

### CODE REVIEW

File: src/components/userInfo.tsx [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  userName: string;
+  userImg: string;
+};
+
+export function UserInfo({ userName, userImg }: userInfoProps): JSX.Element {
+  return (
+    <div className="flex flex-row">
+      <div className="flex flex-col gap-3">
+        <h1 className="text-body">Welcome {userName}!</h1>
+        <p className="text-title">Let’s relax and watch a movie!</p>
```
Comment (ReallyBoard123) [inline src/components/userInfo.tsx:13]: the unicode for the hand symbol is missing

File: src/components/userInfo.tsx [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import React from "react";
+
+type userInfoProps = {
+  userName: string;
+  userImg: string;
+};
+
+export function UserInfo({ userName, userImg }: userInfoProps): JSX.Element {
+  return (
+    <div className="flex flex-row">
```
Comment (ReallyBoard123) [inline src/components/userInfo.tsx:10]: flex by default is always row, so need not add that

**Review by ReallyBoard123 — CHANGES_REQUESTED** [review]
when I rendered it out on storybook, the avatar of the profile picture seems to be placed differently as it is visible on the figma file. Please make a quick reference to it. Otherwise good job! Not many changes to go!!  👋

File: src/components/userInfo.tsx (hunk 1 above)
Comment (kikavlntn) [reply src/components/userInfo.tsx:13]: ups

**Review by ReallyBoard123 — APPROVED** [review]
Wohoo!!

**Pull request merged**